README.md

```diff
@@ -151,12 +151,12 @@ docker_compose.yml files, IE:
                 -d dperson/samba -p
 
 * Attempting to connect with the `smbclient` commandline tool. By default samba
-still tries to use SMB1, which is depriciated and has security issues. This
-container defaults to SMB2, which for no decernable reason even though it's
-supported is disabled by default so run the command as `smbclient -m SMB3`, then
+still tries to use SMB1, which is deprecated and has security issues. This
+container defaults to SMB2, which for no discernible reason even though it's
```

Comment:
Loving the grammar fixes to improve readability.

Nit: Also change "it's" to "its". Thanks.

Reply:
Hoppla. Done 👍